packages/cli/config/jest.js

```diff
@@ -194,9 +194,16 @@ async function getProjectConfig(targetPath, extraConfig) {
     ...getRoleConfig(closestPkgJson?.backstage?.role),
   };
 
+  options.setupFilesAfterEnv = [];
```

Comment:
Feeling we should `options.setupFilesAfterEnv = options.setupFilesAfterEnv || []` here